openwisp_monitoring/monitoring/migrations/0001_squashed_0023_alert_settings_tolerance_remove_default.py

```diff
@@ -48,9 +49,11 @@ class Migration(migrations.Migration):
 
     dependencies = [
         ('monitoring', '__first__'),
```

Comment:
I wonder whether we can remove this as well, seems to me we should be able to, can you try @yashikajotwani12?

Reply:
Yeah! Sure I would remove these and check if everything works fine.

Reply:
![Screenshot from 2022-01-06 22-34-55](https://user-images.githubusercontent.com/77020164/148421902-1c5512d2-3c47-4201-9e10-0f6871631606.png)
I am not getting any errors while removing these, @nemesisdesign

Reply:
@yashikajotwani12 not seeing any error may not mean the code is bug free. I will suggest to try accessing some endpoints too. Verify if they works as expected

Reply:
Try to create a fresh database, running migration command on it and then verifying it

Reply:
> @yashikajotwani12 not seeing any error may not mean the code is bug free. I will suggest to try accessing some endpoints too. Verify if they works as expected

okay! I would try and verify. 👍🏻